introduction.md

```diff
@@ -1,3 +1,8 @@
 Hello, dojo. I'm the other man
 
 My favorite path is Ruby
+
+* Git
+* Ruby
+* Javascript
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?